query/query.go

```diff
@@ -249,6 +266,12 @@ type BranchRepos struct {
 	Repos  *roaring.Bitmap
 }
 
+// Similar to BranchRepos but will be used to match only by repoid and
+// therefore matches all branches
+type RepoIds struct {
```

Comment:
nit: rename try use ID capitialized in all cases you have added code here. See reference here https://github.com/golang/go/wiki/CodeReviewComments#initialisms

Reply:
Done